### DISCUSSION TITLE
[filament/admin] Add ability to specify widgets' column span individually

### DISCUSSION THREAD

**Discussion by detygon** [filamentphp/filament#900] [Ideas]
It would be great to be able to specify widgets column span individually like `filament/forms. This is expecially useful for `ChartWidget`s.

**Comment by danharrin** [comment]
You can do this by adding `col-span-x` to the outer element of your widget view :)

**Comment by detygon** [comment]
Thanks for your quick answer, actually I was using the provided template so I was not able to customize the view.
